src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -482,6 +508,14 @@ private bool TryGetWorkspaceFixersPriorityMap(TextDocument document, [NotNullWhe
 
                     foreach (var (span, diagnostics) in fixerToRangesAndDiagnostics[fixer])
                     {
+                        // Log an individual telemetry event for slow codefix computations to
+                        // allow targeted trace notifications for further investigation. 500 ms seemed like
+                        // a good value so as to not be too noisy, but if fired, indicates a potential
+                        // area requiring investigation.
+                        const int CodeFixTelemetryDelay = 500;
+
+                        using var _ = TelemetryLogging.LogBlockTime(FunctionId.CodeFix_Delay, fixer.GetType().Name, CodeFixTelemetryDelay);
```

Comment:
@mavasani i can't remember.  was Get().Name costly? didn't we need to cache this somewherE?

Reply:
It's used inside the code below. I can put it in a local and have both places use it instead of calling it twice if that would appease the perf issue.